activerecord/lib/active_record/relation.rb

```diff
@@ -539,8 +539,9 @@ def touch_all(*names, time: nil)
     # Destroys the records by instantiating each
     # record and calling its {#destroy}[rdoc-ref:Persistence#destroy] method.
     # Each object's callbacks are executed (including <tt>:dependent</tt> association options).
-    # Returns the collection of objects that were destroyed; each will be frozen, to
-    # reflect that no changes should be made (since they can't be persisted).
+    # Returns the collection of objects that were destroyed if
+    # +config.active_resource.destroy_all_in_batches+ is set to +false+. Each
```

Comment:
```suggestion
    # +config.active_record.destroy_all_in_batches+ is set to +false+. Each
```

This typo makes it obvious that this patch is based on https://github.com/rails/rails/pull/22513. You should credit the original author for their work.

Reply:
Yeah that's true, forgot to mention that in the description updated thanks @eugeneius 🙇